src/scripts/StorePouchDB.js

```diff
@@ -35,8 +12,10 @@ class StoreScrapeResultWriter {
 		if (docs.length === 0) {
 			return Promise.resolve();
 		}
-		const normalizedDocs = normalizeProperties(docs);
-		return this.db.bulkDocs({ docs: normalizedDocs }).catch(err => {
+		// Extra wrapping is added since PouchDB forbids identifiers starting
+		// with underscore. We use them for our attachments and meta information.
+		// TODO Consider moving custom properties from doc to wrapper level
+		return this.db.bulkDocs({ docs: docs.map(doc => ({ data: doc })) }).catch(err => {
```

Comment:
Do we need { data: doc } in parentheses?

Reply:
Yes, that's how you [return object literal in arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Returning_object_literals)